.gitpod.yml

```diff
@@ -0,0 +1,18 @@
+# List the start up tasks. Learn more https://www.gitpod.io/docs/config-start-tasks/
+tasks:
+  - name: yarn
+    init: yarn install
+    command: yarn run start
+# List the ports to expose. Learn more https://www.gitpod.io/docs/config-ports/
+ports:
+  - port: 3000
+    onOpen: open-browser
+github:
```

Comment:
I believe enabling this might exhaust the future configu Gitpod account plan, correct? I've recently opened one and applied to the OSS program, but approval might take a bit. In the meantime, I'd suggest we only prebuild the master branch. Here are some related links for reference:

[Gitpod Open Source Program](https://www.gitpod.io/discover/opensource)
[Gitpod Pricing](https://www.gitpod.io/pricing)

Reply:
I always run it on free account and i dont think it would eat through the gitpid plan for open ports as they are free only multi tenenting and hosting on gitpod is chargeable.

Reply:
I was talking on the prebuilds configuration

Reply:
So should i remove the port as of now?

Reply:
please see the changes I made to the .gitpod file